invoker/rh-changes/pom.xml

```diff
@@ -82,6 +82,12 @@
                 <version>${redhat.che.version}</version>
                 <type>war</type>
             </dependency>
+            <dependency>
+                <groupId>com.redhat.che</groupId>
+                <artifactId>fabric8-ide-stacks</artifactId>
+                <version>${redhat.che.version}</version>
```

Comment:
@davidfestal we should consider it as a customization of upstream che. So it makes sense to use upstream version. 

But I do think that since it's a customization of a `json` file it could be implemented as a patch rather than a maven module (c.f. [my comment](https://github.com/redhat-developer/rh-che/pull/54#issuecomment-298862849)).